prboom2/src/SDL/i_video.c

```diff
@@ -1278,7 +1278,8 @@ void I_UpdateVideoMode(void)
     sdl_renderer = SDL_CreateRenderer(sdl_window, -1, flags);
 
     // [FG] aspect ratio correction for the canonical video modes
-    if (SCREENHEIGHT == 200 || SCREENHEIGHT == 400)
+    if ((SCREENWIDTH == 320 && SCREENHEIGHT == 200) ||
```

Comment:
It's intended that this doesn't check width because it's supposed to support widescreen variants of the low res mode.

Reply:
A custom widescreen resolution will generally have square pixels, so this will just cause unexpected distortion.  I can drop it from the PR since it's unrelated.